src/components/Form/VirtualDropdown.js

```diff
@@ -115,7 +116,7 @@ const renderDropdown = ({label, focus, white, black, items, onFocus, onBlur}) =>
     </LButton>}
     <Inner isOpen={isOpen}>
       <Label top={!!selectedItem || focus} focus={isOpen || focus} text={label} white={white && !isOpen} black={black || (white && isOpen)}>
-        <LButton {...getToggleButtonProps()} onFocus={onFocus} onBlur={onBlur} white={white && !isOpen} black={black || (white && isOpen)}>
+        <LButton style={{lineHeight: pxToRem(22)}} {...getToggleButtonProps()} onFocus={onFocus} onBlur={onBlur} white={white && !isOpen} black={black || (white && isOpen)}>
```

Comment:
how is this `LButton` a special case that warrants inline style instead of changing the source?